app/views/projects/blob/new.html.haml

```diff
@@ -1,5 +1,11 @@
-- page_title "New File", @ref
-%h3.page-title New file
+%h3.page-title<
+  Create new file or&nbsp;
+  %a{href: '#modal-upload-blob', 'data-target' => '#modal-upload-blob', 'data-toggle' => 'modal'}>upload existing one
```

Comment:
I changed the color to blue, it seems more obvious.
![color](https://cloud.githubusercontent.com/assets/7370334/9855867/4c628424-5b11-11e5-8f8f-5f04377d5971.png)